frontend_app_workspace/src/container/PopupCreateWorkspace.jsx

```diff
@@ -169,7 +169,11 @@ export class PopupCreateWorkspace extends React.Component {
     ))
 
     switch (fetchPostSpace.apiResponse.status) {
-      case 200: this.handleClose(); break
+      case 200:
+        props.data.config.history.push(props.data.config.PAGE.WORKSPACE.DASHBOARD(fetchPostSpace.body.workspace_id))
```

Comment:
Don't you have a direct access to `history`: `props.history`?

Reply:
unfortunately not, we are not in `frontend/`and don't have redux here.